docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+===================
+Instructors’ Guide
+===================
+
```

Comment:
please activate the table of contents with 2 toc depth here

Reply:
The TOC is visible now on the left side, consistent with the rest of the documentation. Is this ok? 
![image](https://user-images.githubusercontent.com/15110960/96521301-3b464f00-1271-11eb-88d3-a566c9e50c2a.png)

Sphinx does not create a TOC for one file, it constructs the TOC from a tree. If we want to have a TOC leading the instructor guide we would have to divide all sections into individual files.